week1/homework/out/Contact.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+exports.Contact = class Contact {
+	constructor(name, phone) {
+		this.name = name;
+		this.phone = phone;
+	}
+
+	get phone() {
+		return this._phone;
+	}
+	set phone(num) {
+		// convert string to int if it's valid number 
+		if (typeof num === 'string') {
+			// convert string to array
+			let a = num.split('');
```

Comment:
good you write some comments to explain what's happening! however, `a`, `x`, etc. don't mean anything. Give your variables names that mean something, like `let phoneNumberArray = num.split('');`